js/kraken.js

```diff
@@ -209,9 +209,16 @@ module.exports = class kraken extends Exchange {
     }
 
     async fetchMinOrderSizes () {
-        this.parseJsonResponse = false;
-        let html = await this.zendeskGet205893708WhatIsTheMinimumOrderSize ();
-        this.parseJsonResponse = true;
+        let html;
```

Comment:
this one is missing an undefined rhs value )

Reply:
Let me know, if you want me to fix it.

Reply:
Nope. I'll fix the linter.

Reply:
Ok, fixed. Let me know if you want separate linter commits for such cases.

Reply:
I'm ok if they go in the same pr, as they are related anyways, and they are too small to detach them, I think.